<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mazegen</class>
 <widget class="QMainWindow" name="mazegen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mazegen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Bstart">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>460</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>70</y>
      <width>254</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="BTview">
       <property name="text">
        <string>阅览</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTsav">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTsvans">
       <property name="text">
        <string>保存答案</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>随机 dfs</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="LWarn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>176</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="3" column="1">
      <widget class="QLineEdit" name="Im"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="LsT">
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>地图设定</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Ln">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>行数(n)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="In"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Lm">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>列数(m)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Lsz">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>单元格大小</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="Isz"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>280</y>
      <width>132</width>
      <height>138</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="LM_DT">
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>路径引导-权重</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="LM_Dd">
       <property name="text">
        <string>↓</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="IM_Dd">
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="LM_Dr">
       <property name="text">
        <string>→</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="IM_Dr">
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="LM_Du">
       <property name="text">
        <string>↑</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="IM_Du">
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="LM_Dl">
       <property name="text">
        <string>← </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="IM_Dl">
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="ptpi">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>391</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
